phpseclib/Crypt/Common/SymmetricKey.php

```diff
@@ -840,6 +849,21 @@ public function encrypt($plaintext)
                     return $result;
                 case self::MODE_CTR:
                     return $this->openssl_ctr_process($plaintext, $this->encryptIV, $this->enbuffer);
+                case self::MODE_IGE:
+                    $ciphertext = '';
```

Comment:
Yes, but I wasn't sure whether it was a good idea to create a separate function just to avoid duplicating a few lines of code. 
Should I do that?